tests/resources/dsl/botched_deployment.yaml

```diff
@@ -0,0 +1,11 @@
+tosca_definitions_version: cloudify_dsl_1_2
```

Comment:
i'm thinking there must have already been a blueprint identical to this, isnt there?
if so i'd probably use that one and just have the addition in the code for the flag to raise an error from the operation
not sure the duplication of the yaml is needed